app/src/main/java/dev/arkbuilders/arkmemo/ui/fragments/EditGraphicNotesFragment.kt

```diff
@@ -77,22 +69,25 @@ class EditGraphicNotesFragment: Fragment(R.layout.fragment_edit_notes) {
         activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
         activity.showSettingsButton(false)
 
-        noteTitle.hint = defaultTitle
+        noteTitle.hint = getString(R.string.hint_new_graphical_note)
         noteTitle.setText(title)
         noteTitle.addTextChangedListener(noteTitleChangeListener)
         notesCanvas.isVisible = true
         notesCanvas.setViewModel(graphicNotesViewModel)
         saveButton.setOnClickListener {
             val svg = graphicNotesViewModel.svg()
             val note = GraphicNote(
-                title = title.ifEmpty { defaultTitle },
+                title = binding.noteTitle.text.toString(),
```

Comment:
variable ```title``` should be used here

Reply:
@ShubertMunthali Can you elaborate further?
`binding.noteTitle.text.toString()` is basically the same as `title` unless user enters his own title.

Reply:
It's a little confusing where the title is coming from :). My first glance at that line I thought the title is missing :)